src/notes/notes_controller.ts

```diff
@@ -0,0 +1,22 @@
+import { getAllPredefinedNotes } from "./get_all_notes";
+import {Request, Response} from 'express';
+import { getPredifinedUsers } from "./get_all_users";
+
+export class NotesController{
```

Comment:
This is a good idea,  maybe you can migrate the functions in crudNotes.ts into this file and then update the unit tests as well.

Reply:
But the CrudNotes has all the functions which are related to AWS also,thats why did my demo type controller in seperate class, like is it required to move or migrate code and make change in tests also..